ompi/mca/coll/tuned/coll_tuned_module.c

```diff
@@ -91,6 +92,11 @@ ompi_coll_tuned_comm_query(struct ompi_communicator_t *comm, int *priority)
     tuned_module->super.coll_module_enable = tuned_module_enable;
     tuned_module->super.ft_event = mca_coll_tuned_ft_event;
 
+#if OPAL_ENABLE_FT_MPI
```

Comment:
Are we sure this always return a static strucure? The rest of the logic does mark them as NULL (for example alltoallw just below). That would be indeed nice to avoid the spillover.

Reply:
tuned does (it set everything to NULL based on COLLCOUNT (so as long as these new collectives are part of the COLLTYPE enum, it should work. LEt's ignore this for now, we have more important things to address.